tests/frontend/pages/wallets/wallets.py

```diff
@@ -28,6 +28,42 @@ def create(self, name):
         wait = WebDriverWait(self.driver, 60)
         wait.until(EC.invisibility_of_element_located((By.CLASS_NAME, "v-dialog")))
 
+    def add_funded(self, name):
+        # get funded wallet button and click
+        buttons = self.driver.find_elements_by_class_name("v-btn")
+        add_funded_button = [button for button in buttons if button.text == "ADD TESTNET FUNDED WALLET"][0]
```

Comment:
this will fail because on merging: https://github.com/threefoldtech/js-sdk/pull/1709

the 2 should adapted then, or should adapt the other PR ?